Assets/Makefile.am

```diff
@@ -1,5 +1,8 @@
-assetsdir = $(datadir)/ObjectivelyMVC
+%.h: %
+	xxd -i $^ $@
 
-assets_DATA = \
-	check.png \
-	resize.png
\ No newline at end of file
+BUILT_SOURCES = \
+	check.png.h \
+	debug.json.h \
+	resize.png.h \
+	stylesheet.css.h
```

Comment:
@Paril Currently, I'm versioning the "baked" headers for these assets. It's not ideal, but it's also not the end of the world. If you have an `xxd` equivalent on Windows, tho, I would like to not version them, so please let me know.